src/main/webapp/styles/add-recommendation.css

```diff
@@ -87,7 +87,7 @@ input[type="submit"] {
   padding: 12px 20px;
 }
 
-/*Formatting for Radio Buttons */
+/* --- Radio Buttons for Add Rec Form --- */
```

Comment:
please be consistent whether you want to add blank line afterthe comment
also, may I know why do we need "---"